src/types/Tooltip/index.tsx

```diff
@@ -23,6 +23,8 @@ type TooltipType = {
   trigger: triggerType;
   // Align tooltip with respect to the target
   align: alignType;
+  //Adding style to tooltip content
+  className: any;
```

Comment:
```suggestion
  // Adding style to tooltip content
  className: string;
```
Try to avoid using `any` for any typing if you can help it. In this case, className is exclusively a string so it's a better type to use.